lib/logzio-nodejs.d.ts

```diff
@@ -0,0 +1,30 @@
+type CallbackFunction = (err: Error, bulk: object) => void;
+
+interface ILoggerOptions {
+    token: string;
+    host?: string;
+    type? : string;
+    sendIntervalMs? : number;
+    bufferSize? : number;
+    debug? : boolean;
+    numberOfRetries? : number;
+    supressErrors?  : boolean;
+    addTimestampWithNanoSecs? : boolean;
+    compress? : boolean;
+    internalLogger : object;
+    protocol? : string;
+    port : string;
+    timeout? : number;
+    sleepUntilNextRetry? : number;
+    callback(err: Error, bulk: object): void;
```

Comment:
This actually looks like the `CallbackFunction`.
Maybe call it `LoggerCallback` ?

Reply:
I rather not change names in the logger so it won't effect users.